### PR TITLE
[Dialogs] Flash scroll indicators if content is larger than the scroll view.

### LINKED ISSUE #7405: [Dialogs] Internal issue: b/132900320
This was filed as an internal issue. If you are a Googler, please visit [b/132900320](http://b/132900320) for more details.

---
#### Internal data
- Associated internal bug: [b/132900320](http://b/132900320)

### PR DESCRIPTION
**Pull request by codeman7**
## Motivation
At times a _Dialog_ can contain information that doesn't fit within the UIScrollViews. This causes users to not know that there is scrollable content within that scrollView.

## Detailed changes
Inspect each scrollView to see if its `bounds.height` is greater than the `contentSize.height` if it is then `flashScrollIndicators` on the respective UIScrollView. We do not need to check the width because we control the UIScrollViews and make them grow larger as content gets added. 

We may need to do a follow up to `flashScrollIndicators` within `viewWillTransitionToSize:` if that's needed, for now I believe this closes #7405.

Closes #7405 

## Screenshots
| Before | After |
| - | - | 
|Simulator Screen Shot - iPhone 7 - 2019-05-21 at 15 40 16|Simulator Screen Shot - iPhone 7 - 2019-05-21 at 15 41 29|

### CODE REVIEW

File: components/Dialogs/src/MDCAlertController.m [hunk 1]
```diff
@@ -465,6 +465,20 @@ - (void)viewWillTransitionToSize:(CGSize)size
...
+- (void)viewDidAppear:(BOOL)animated {
+  [super viewDidAppear:animated];
+
+  if (self.alertView.contentScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.contentScrollView.frame)) {
+    [self.alertView.contentScrollView flashScrollIndicators];
+  }
+
+  if (self.alertView.actionsScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.actionsScrollView.frame)) {
```
Comment (romoore) [inline components/Dialogs/src/MDCAlertController.m]: Should these be the `bounds` rather than the `frame`?

**Review by romoore — APPROVED** [review]

File: components/Dialogs/src/MDCAlertController.m [hunk 2]
```diff
@@ -465,6 +465,20 @@ - (void)viewWillTransitionToSize:(CGSize)size
...
+
+  if (self.alertView.contentScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.contentScrollView.bounds)) {
+    [self.alertView.contentScrollView flashScrollIndicators];
+  }
+
+  if (self.alertView.actionsScrollView.contentSize.height >
+      CGRectGetHeight(self.alertView.actionsScrollView.bounds)) {
+    [self.alertView.actionsScrollView flashScrollIndicators];
+  }
```
Comment (jverkoey) [inline components/Dialogs/src/MDCAlertController.m]: I think you don't need these checks; you should just be able to flash the scroll indicators and UIKit will determine whether it needs to.

File: components/Dialogs/src/MDCAlertController.m (hunk 1 above)
Comment (jverkoey) [reply components/Dialogs/src/MDCAlertController.m]: Yep they should be, assuming we need these checks (see my previous comment).

File: components/Dialogs/src/MDCAlertController.m (hunk 2 above)
Comment (codeman7) [reply components/Dialogs/src/MDCAlertController.m]: Good call! Updated the PR.

**Pull request merged**